src/adaptors/articles.ts

```diff
@@ -0,0 +1,13 @@
+import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
+
+export default interface Articles extends Iterable<JsonLdObj> {
```

Comment:
Following https://github.com/libero/publisher/issues/346, this should probably be a [`Store`](https://rdf.js.org/stream-spec/#store-interface)...